packages/core/src/addresses/staging.ts

```diff
@@ -66,7 +66,7 @@ export const addresses: Addresses = {
         l2AmmWrapper: '0xCb5DDFb8D0038247Dc0bEeeCAa7f3457bEFcb77c',
         l2SaddleSwap: '0x4AD8db323F6EBEC4DEb53140FfC7dDb22DE5f607',
         l2SaddleLpToken: '0xC37fA2448eb8ddd54e50D11D9fF4b82F1D01d7Dc',
-        l1FxBaseRootTunnel: '0x3c0FFAca566fCcfD9Cc95139FEF6CBA143795963',
+        l1FxBaseRootTunnel: '0x7A08f1409E5D3e9Bc49063aA4009fB4BBC8777C3',
```

Comment:
Can you change the `l1MessengerWrapper` to this `0x7A08f1409E5D3e9Bc49063aA4009fB4BBC8777C3` as well? For Polygon `l1FxBaseRootTunnel` and `l1MessengerWrapper` are the same address.

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/384/commits/fef0c0adff680b369340f9b219ca3b5acbd002b8